nucypher/policy/collections.py

```diff
@@ -186,6 +186,20 @@ def public_id(self) -> str:
         """
         return self._id
 
+    @classmethod
+    def from_obj_or_bytes(cls, obj):
+
+        if isinstance(obj, cls):
+            return obj
+
+        # TODO: This LBYL is ugly and fraught with danger.  NRN
```

Comment:
Let's track this with an Issue.

Reply:
Funny thing, I actually can't remember now why I needed this method. It's not used anywhere. Seems to have been partially based on the bit in the beginning of `retrieve()`, and that's where the TODO comes from, too. Don't even know what LBYL means. 

Deleted it completely.